proxy/src/proxy/mod.rs

```diff
@@ -176,6 +177,9 @@ pub(crate) fn finish_client_init(
         });
     }
 
+    // Expose session_id to clients
+    client.write_message(BeMessage::NoticeResponse(&ctx.session_id().to_string()));
```

Comment:
Considering this is often displayed in logs and in psql by default, maybe we should make this a bit more human friendly?

I also want to move us to replacing our uuids with https://github.com/jetify-com/typeid (my rust impl https://docs.rs/type-safe-id/latest/type_safe_id/). They're more compact and have a nice identitier which makes it easy when someone just pastes you an ID. I'm imagining neon_conn_abc...

Reply:
Yes, I remember that probably we even discussed it some time ago

Reply:
```
NOTICE:  Neon session_id: d4562a11-f75c-41a2-85b7-dc5d44847468

psql (17.5)
SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off, ALPN: postgresql)
Type "help" for help.

postgres=#
```